tests/wasm_tests/wasm_tests.cpp

```diff
@@ -226,44 +202,105 @@ BOOST_FIXTURE_TEST_CASE( abi_from_variant, tester ) try {
 
 } FC_LOG_AND_RETHROW() /// prove_mem_reset
 
-BOOST_FIXTURE_TEST_CASE( test_api_bootstrap, tester ) try {
+BOOST_FIXTURE_TEST_CASE( test_deferred_failure, tester ) try {
```

Comment:
pretty sure this test was moved to the currency tests as it required currency... is this file disabled from running?